Packages/vcs/Lib/utils.py

```diff
@@ -1716,6 +1716,37 @@ def getworldcoordinates(gm, X, Y):
     return wc
 
 
+def rgba_color(color, colormap):
+    """Try all of the various syntaxes of colors and return 0-100 RGBA values."""
+    try:
+        # Is it a colormap index?
+        return colormap.index[color]
+    except ValueError:
```

Comment:
Nope, it's a `ValueError`. We implement `__setitem__` and `__getitem__` on the colormap object, and raise a `ValueError` if the key isn't a number between 0 and 256.